litmus-portal/frontend/src/components/LocalQuickActionCard/index.tsx

```diff
@@ -97,7 +97,7 @@ const LocalQuickActionCard: React.FC<LocalQuickActionCardProps> = ({
       ? {
           src: '/icons/survey.svg',
           alt: 'survey',
-          onClick: () => window.open('https://forms.gle/qMuVphRyEWCFqjD56'),
+          onClick: () => window.open('https://forms.gle/KQp5qj8MUneMSxLp9'),
```

Comment:
@S-ayanide can we use this from config file ? changing it would be easier for the community?

Reply:
Sure, have made a `constants.ts` file at the root